scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala

```diff
@@ -82,6 +83,14 @@ class HtmlRenderer(rootPackage: Member, members: Map[DRI, Member])(using ctx: Do
       linkResources(page.link.dri, earlyResources, deferJs = false).toList,
       linkResources(page.link.dri, resources, deferJs = true).toList,
       script(raw(s"""var pathToRoot = "${pathToRoot(page.link.dri)}";""")),
+      (page.content match
+        case ResolvedTemplate(loadedTemplate, _) =>
+          val path = loadedTemplate.file.toPath
+          ctx.sourceLinks.repoSummary(path) match
+            case Some(DefinedRepoSummary("github", org, repo)) =>
+              script(raw(s"""var githubContributorsUrl = "https://api.github.com/repos/$org/$repo/commits?path=docs/${relativePath(path).toString.stripPrefix("docs-for-dotty-page/")}";"""))
```

Comment:
I think it's not an issue this time, but I will consider changing it to json or something like that

Reply:
Actually, JSON here would not be so effective, since we embed the exact link at "compile time". Each page has its own URL, hence I guess it's better to leave it as it is now.